### PR TITLE
Nav link to

### PR DESCRIPTION
**Pull request by matt-riemer**

### CODE REVIEW

**Comment by alanvardy** [comment]
Cool! So do we need to change the `link` functions to `nav_link`?

**Comment by matt-riemer** [comment]
I will add test coverage for this once it's merged

File: lib/health_web/templates/layout/_navbar.html.eex [hunk 1]
```diff
@@ -37,10 +36,10 @@
       </li>
       <% else %>
       <li class="nav-item">
-        <%= link "Register", to: Routes.pow_registration_path(@conn, :new), class: "nav-link" %>
```
Comment (alanvardy) [inline lib/health_web/templates/layout/_navbar.html.eex]: Shouldn't this be nav_link? Or are you hoping to make that change later?

Comment (matt-riemer) [reply lib/health_web/templates/layout/_navbar.html.eex]: This should ideally be `nav_link` or I can make the change later

**Pull request merged**